<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartupDialog</class>
 <widget class="QDialog" name="StartupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Map</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QLineEdit" name="MapDirEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>RandomMap</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="MapEditButton">
          <property name="text">
           <string>MapEdit</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>テクスチャ：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>あっさり</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>こってり</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="ClientSettingForm" name="HotGroupBox">
       <property name="title">
        <string>HOT</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QPushButton" name="ServerStartButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>ゲーム開始</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="ClientSettingForm" name="CoolGroupBox">
       <property name="title">
        <string>COOL</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" rowspan="2" colspan="2">
      <widget class="QGroupBox" name="ServerGroup">
       <property name="title">
        <string>SERVER</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="1">
           <widget class="QLabel" name="GrobalIPLabel">
            <property name="text">
             <string>不明</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="LocalIPLabel">
            <property name="text">
             <string>192.168.1.1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>グローバルIP:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>ローカルIP:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>ホスト名</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="HostName">
            <property name="text">
             <string>不明</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>GameMusic：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="GameMusicCombo">
            <item>
             <property name="text">
              <string>TwinBeeNew</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Necromancer</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>ServerSetting</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClientSettingForm</class>
   <extends>QGroupBox</extends>
   <header>ClientSettingForm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ServerStartButton</sender>
   <signal>clicked()</signal>
   <receiver>StartupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>StartupDialog</receiver>
   <slot>PushedMapSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MapEditButton</sender>
   <signal>clicked()</signal>
   <receiver>StartupDialog</receiver>
   <slot>ShowMapEditDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>StartupDialog</receiver>
   <slot>ChangedTexture(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>StartupDialog</receiver>
   <slot>Setting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GameMusicCombo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>StartupDialog</receiver>
   <slot>ChangeMusicCombo(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CoolConnectionToggled(bool)</slot>
  <slot>HotConnectionToggled(bool)</slot>
  <slot>CoolComboBoxChenged(QString)</slot>
  <slot>HotComboBoxChenged(QString)</slot>
  <slot>PushedMapSelect()</slot>
  <slot>ShowMapEditDialog()</slot>
  <slot>ChangedTexture(QString)</slot>
  <slot>Setting()</slot>
  <slot>ChangeMusicCombo(QString)</slot>
 </slots>
</ui>
